code/__HELPERS/roundend.dm

```diff
@@ -0,0 +1,412 @@
+/datum/controller/subsystem/ticker/proc/gather_roundend_feedback()
+	var/clients = GLOB.player_list.len
+	var/surviving_humans = 0
+	var/surviving_total = 0
+	var/ghosts = 0
+	var/escaped_humans = 0
+	var/escaped_total = 0
+
+	for(var/mob/M in GLOB.player_list)
+		if(ishuman(M))
+			if(!M.stat)
+				surviving_humans++
+				if(M.z == ZLEVEL_CENTCOM)
+					escaped_humans++
+		if(!M.stat)
+			surviving_total++
+			if(M.z == ZLEVEL_CENTCOM)
+				escaped_total++
+
+		if(isobserver(M))
+			ghosts++
+
+	if(clients)
+		SSblackbox.record_feedback("nested tally", "round_end_stats", clients, list("clients"))
+	if(ghosts)
+		SSblackbox.record_feedback("nested tally", "round_end_stats", ghosts, list("ghosts"))
+	if(surviving_humans)
+		SSblackbox.record_feedback("nested tally", "round_end_stats", surviving_humans, list("survivors", "human"))
+	if(surviving_total)
+		SSblackbox.record_feedback("nested tally", "round_end_stats", surviving_total, list("survivors", "total"))
+	if(escaped_humans)
+		SSblackbox.record_feedback("nested tally", "round_end_stats", escaped_humans, list("escapees", "human"))
+	if(escaped_total)
+		SSblackbox.record_feedback("nested tally", "round_end_stats", escaped_total, list("escapees", "total"))
+
+	gather_antag_success_rate()
+
+/datum/controller/subsystem/ticker/proc/gather_antag_success_rate()
+	var/team_gid = 1
+	var/list/team_ids = list()
+
+	for(var/datum/antagonist/A in GLOB.antagonists)
+		var/list/antag_info = list()
+		antag_info["ckey"] = A.owner.key
```

Comment:
one of these things is not like the other?

Reply:
You mean the key/ckey or what ? Minds should be changed to use ckey instead of key but that's separate issue.